Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -523,8 +523,6 @@ class SettingsTableViewController: ThemedTableViewController {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
-        settings = generateSettings()
```

Comment:
`generateSettings` is called way too many times. We should debug it further so it will be called only once.